Stop relying on javax.json 1.1.4

### Describe the bug (required)

I got a a build error saying

```
[ERROR] Failed to execute goal io.github.git-commit-id:git-commit-id-maven-plugin:5.0.0:revision
....
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact jakarta.json:jakarta.json-api:jar:1.1.4
```

I cloned your repo to check the dependencies (i.e. running mvn dependency ) and I noticed  there is a reference to 
 +- javax.json:javax.json-api:jar:1.1.4:compile
not sure why build tried to look for jakarta.json 1.1.4 but in any case, javax... is now replaced with jakarta...

Could you please update your package dependency to jakarta.json-api instead . 
seems the earliest version of jakarta.json-api is 1.1.5
https://mvnrepository.com/artifact/jakarta.json/jakarta.json-api



### Tell us about your plugin configuration (required)

```xml
            <plugin>
                <groupId>io.github.git-commit-id</groupId>
                <artifactId>git-commit-id-maven-plugin</artifactId>
                <version>5.0.0</version>
                <executions>
                    <execution>
                        <id>get-the-git-infos</id>
                        <goals>
                            <goal>revision</goal>
                        </goals>
                        <phase>initialize</phase>
                    </execution>
                </executions>
                <configuration>
                    <generateGitPropertiesFile>true</generateGitPropertiesFile>
                    <generateGitPropertiesFilename>${project.build.outputDirectory}/git.info</generateGitPropertiesFilename>
                    <includeOnlyProperties>
                        <includeOnlyProperty>^git.branch$</includeOnlyProperty>
                        <includeOnlyProperty>^git.commit.time$</includeOnlyProperty>
                        <includeOnlyProperty>^git.build.(time|version)$</includeOnlyProperty>
                        <includeOnlyProperty>^git.commit.id.(abbrev|full)$</includeOnlyProperty>
                        <includeOnlyProperty>^git.local.branch.(ahead|behind)$</includeOnlyProperty>
                    </includeOnlyProperties>
                    <commitIdGenerationMode>full</commitIdGenerationMode>
                </configuration>
            </plugin>
```


### Tell us about the Plugin version used (required)

5

### Tell us about the Maven version used (required)

```bash
mvn --version
// Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63)
```


### Steps to Reproduce (required)

I run the plugin with ``mvn clean deploy`` VS ``mvn deploy:deploy``...


### Are there any stacktraces or any error messages? (required)

**WARNING** You may **NOT** want to paste all the output that is produced with `verbose` publicly,
since it **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```bash
mvn clean deploy
// ????
```


### Is there a (public) project where this issue can be reproduced? (optional)

_No response_

### Your Environment (optional)

_No response_

### Context (optional)

_No response_

--- a/pom.xml
+++ b/pom.xml
@@ -196,14 +196,14 @@
         </dependency>
         <!-- json stuff -->
         <dependency>
-            <groupId>javax.json</groupId>
-            <artifactId>javax.json-api</artifactId>
-            <version>1.1.4</version>
+            <groupId>jakarta.json</groupId>
+            <artifactId>jakarta.json-api</artifactId>
+            <version>2.1.2</version>
         </dependency>
         <dependency>
             <groupId>org.glassfish</groupId>
-            <artifactId>javax.json</artifactId>
-            <version>1.1.4</version>
+            <artifactId>jakarta.json</artifactId>
+            <version>2.0.1</version>
         </dependency>
         <!-- yaml -->
         <dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -201,9 +201,9 @@
             <version>2.1.2</version>
         </dependency>
         <dependency>
-            <groupId>org.glassfish</groupId>
-            <artifactId>jakarta.json</artifactId>
-            <version>2.0.1</version>
+            <groupId>org.eclipse.parsson</groupId>
+            <artifactId>parsson</artifactId>
+            <version>1.1.4</version>
         </dependency>
         <!-- yaml -->
         <dependency>

--- a/src/main/java/pl/project13/core/util/JsonManager.java
+++ b/src/main/java/pl/project13/core/util/JsonManager.java
@@ -17,15 +17,15 @@
 
 package pl.project13.core.util;
 
+import jakarta.json.Json;
+import jakarta.json.JsonReader;
+import jakarta.json.JsonString;
+import jakarta.json.stream.JsonGenerator;
+import jakarta.json.stream.JsonGeneratorFactory;
 import nu.studer.java.util.OrderedProperties;
 import pl.project13.core.CannotReadFileException;
 
 import javax.annotation.Nonnull;
-import javax.json.Json;
-import javax.json.JsonReader;
-import javax.json.JsonString;
-import javax.json.stream.JsonGenerator;
-import javax.json.stream.JsonGeneratorFactory;
 import java.io.*;
 import java.nio.charset.Charset;
 import java.util.Collections;

--- a/src/test/java/pl/project13/core/util/TestJsonManager.java
+++ b/src/test/java/pl/project13/core/util/TestJsonManager.java
@@ -52,7 +52,6 @@ public class TestJsonManager {
     // then
     Assert.assertEquals(
             Arrays.asList(
-                    "",
                     "{",
                     "    \"git.commit.id\": \"beef4e92e9cabd043b105a14514289f331b40bf2\",",
                     "    \"git.commit.id.abbrev\": \"beef4e9\"",
@@ -101,7 +100,6 @@ public class TestJsonManager {
     // then
     Assert.assertEquals(
             Arrays.asList(
-                    "",
                     "{",
                     "    \"git.commit.id.abbrev\": \"beef4e9\",",
                     "    \"git.commit.id.full\": \"beef4e92e9cabd043b105a14514289f331b40bf2\"",
@@ -150,7 +148,6 @@ public class TestJsonManager {
     // then
     Assert.assertEquals(
             Arrays.asList(
-                    "",
                     "{",
                     "    \"git.commit.message.full\": \"initial commit on test project with some special characters äöüàñ.\",",
                     "    \"git.commit.user.name\": \"Александр Eliáš\"",

